### DISCUSSION TITLE
Automations run only with Significant Delay

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#7293] [Help] [answered]
I have created several on "New Row" and "Update Row" automations. 

- One automation sends an email when a new row is added
- The second automation makes a REST GET request when a new row is added and updates the row with additional data
- The third automation does the same as the first, but on "row update"

When running the tests, the automations work flawlessly. However, once published, the automations only actually execute hours after the respective rows are added or updated. What could be causing this issue? 

I'm using BudibaseDB

**Comment by ghost** [comment]
When I look at my automation history, all of the automations run succesfully, but only a day later at 07:30
![image](https://user-images.githubusercontent.com/7414843/184843128-819c6b49-abfc-4643-b8ae-5405d663ce31.png)

**Comment by melohagan** [comment, marked as answer]
This has been looked at here: https://github.com/Budibase/budibase/issues/7323
